core/scss/components/lockup/_lockup.scss

```diff
@@ -5,120 +5,135 @@
+    @include margin(null -1px null null);
+    @include padding(null 7px 0 null);
+    flex-shrink: 1;
+    align-self: flex-end;
+    max-width: 160px;
+    display: flex;
+    border-right: solid 1px $color-black;
+  }
 
   @include grid-media('md') {
-    @include grid-columns(auto auto 1fr, false);
-    @include grid-rows(auto auto);
+    min-height: 32px;
+  }
+}
+
+.su-lockup__cell2 {
+  @include grid-media('sm') {
+    flex-grow: 1;
+    vertical-align: bottom;
+    align-self: flex-end;
+    border-left: solid 1px $color-black;
+    @include padding(0 null null 7px);
   }
 }
 
 .su-lockup__wordmark {
   @include logo;
-  @include grid-item-spans(1, span 1, 1, span 1);
-  display: block;
   color: $color-cardinal-red;
   font-size: 36px;
-  line-height: 1;
+  line-height: 17px;
+  vertical-align: bottom;
 
   @include grid-media('md') {
     font-size: 46px;
-    height: 33px;
-    line-height: .96;
-  }
-
-  // Hide pipe/horizontal line on smaller devices if there is no line 5
-  + .su-lockup__pipe {
-    display: none;
-
-    @include grid-media('md') {
-      display: block;
-    }
+    line-height: 21px;
   }
 }
 
-//Put the pipe in its own CSS grid cell so that its height auto adjusts to the tallest part of the lockup
-.su-lockup__pipe {
-  @include margin(7px null);
-  @include grid-item-spans(1, span 1, 3, span 1);
-  display: block;
-  content: "";
-  width: 118px;
-  height: 1px;
-  background-color: $color-black;
+.su-lockup__wordmark-wrapper {
+  line-height: 26px;
 
   @include grid-media('md') {
-    @include margin(0 7px);
-    @include grid-item-spans(2, span 1, 1, span 1);
-    display: inline-block;
-    width: 1px;
-    height: 100%;
+    line-height: 32px;
   }
-
 }
 
-.su-lockup__sitename-line1,
-.su-lockup__sitename-line2,
-.su-lockup__sitename-line5 {
+.su-lockup__line1,
+.su-lockup__line2,
+.su-lockup__line3,
+.su-lockup__line4,
+.su-lockup__line5 {
   @include font-smoothing;
-  display: block;
-  @include grid-item-spans(1, span 1);
   color: $color-black;
+  display: block;
 }
 
-
-.su-lockup__sitename-line1 {
-  @include grid-item-spans(null, null, 4, span 1);
-  font-size: 22px;
-  line-height: .8;
-
-  @include grid-media('md') {
-    @include grid-item-spans(3, span 1, 1, span 1);
-    font-size: 30px;
-  }
+.su-lockup__line1 {
+  font-size: 2.8rem;
 }
 
-.su-lockup__sitename-line2 {
-  display: none;
+.su-lockup__line2 {
+  font-size: 2.6rem;
 }
 
-.su-lockup__sitename-line5 {
-  @include grid-item-spans(null, null, 2, span 1);
-  font-size: 20px;
-  line-height: 1.1;
-
-  @include grid-media('md') {
-    @include padding(8px null null);
-    @include grid-item-spans(1, span 3, 2, span 1);
-    font-size: 27px;
-  }
+.su-lockup__line3 {
+  @include margin(0.5rem null null);
+  font-size: 2.6rem;
+  font-style: italic;
 }
 
-.su-lockup__sitename-line1--uppercase,
-.su-lockup__sitename-line5--uppercase {
+.su-lockup__line4 {
+  @include margin(0.5rem null null);
+  font-size: 2.5rem;
```

Comment:
```suggestion
  font-size: 2.5rem;
  letter-spacing: .05rem;
  margin-left: -.2rem;
```

Reply:
Just a small tweak to line up the "Engineering" text with the wordmark if it's needed. If not, please ignore 😺 
![screen shot 2019-03-04 at 3 06 32 pm](https://user-images.githubusercontent.com/42749717/53769176-f600be00-3e8f-11e9-8166-b18c0ea3386f.png)